packages/backend-defaults/config.d.ts

```diff
@@ -121,6 +121,16 @@ export interface Config {
       };
     };
 
+    /**
+     * Options used by the default actions service.
+     */
+    actions?: {
+      /**
+       * List of plugin sources to load actions from.
+       */
+      pluginSources?: string[];
+    };
```

Comment:
We likely want to iterate on this a little bit, but imo let's leave that for followups